graphene_sqlalchemy/tests/test_query.py

```diff
@@ -484,3 +484,36 @@ def makeNodes(nodeList):
             node["node"]["name"] for node in expectedNoSort[key]["edges"]
         )
 
+
+def test_filter(session):
```

Comment:
I suspect this won't get accepted until all of the possible ways to filter are tested.